modules/ROOT/pages/fd-connection-view-audit-logs.adoc

```diff
@@ -0,0 +1,26 @@
+= View Audit Logs for Connections That are Shared
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords:
+
+:imagesdir: ../assets/images
+
+Although only the owner of a shared connection can edit or delete it, someone might illicitly acquire the credentials and edit or delete one or more shared connections. Someone also might maliciously use another user's credentials to create a shared connection and siphon data to an unapproved system.
+
+To help you identify such scenarios, Flow Designer creates an audit-log message for every creation, change to, and deletion of a shared connection within your business group. Each message includes the following information:
+
+* The time and date of the action
+* The product in which the action was taken (i.e., Flow Designer)
+* The type of object audited, which is always "Connection"
+* The type of action: Create, Edit, or Delete
+* The name of the connection
+* The user ID that was used to take the action
+
+You can also download the payload for an entry in the audit log. A payload for the creation, modification, or deletion of a connection includes the ID of the connection, the ID of the user that created the connection, the type of connector that the connection was created for, and more.
+
+== Procedure
```

Comment:
Note inconsistency between using bold inline heads for this and this subtitle.
I think we should avoid inline headings.
I would like to propose gerunds for topic sets like these (Viewing Audit Logs) followed by a short statement and then a titled Concept subtopic and an imperative-titled subtopic (View Audit Logs).